libs/pmp-web/shared/ui-table-statistic/src/lib/table-statistic/table-statistic.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { PrStatistics } from '@pimp-my-pr/shared/domain';
+
+@Component({
+  selector: 'pmp-table-statistic',
+  templateUrl: './table-statistic.component.html',
+  styleUrls: ['./table-statistic.component.scss']
+})
+export class TableStatisticComponent implements OnInit {
+  @Input()
+  tableData: PrStatistics[];
+
+  @Input()
+  columnPropertyName: String;
```

Comment:
Type it as primitive `string`.
https://stackoverflow.com/questions/14727044/typescript-difference-between-string-and-string